types/jsonwebtoken/jsonwebtoken-tests.ts

```diff
@@ -160,12 +160,12 @@ decoded = jwt.decode(token, { complete: false, json: false });
 
 decoded = jwt.decode(token, { json: true });
 if (decoded) {
-    // $ExpectType { [key: string]: any; }
+    // $ExpectType JwtPayload
     decoded;
 }
 
 decoded = jwt.decode(token, { complete: true });
 if (decoded) {
-  // $ExpectType { [key: string]: any; }
+  // $ExpectType Jwt
```

Comment:
This seems to expect `JwtPayload` despite having `complete` set to `true`
I'm not entirely sure why?